registries/_format/date-time.md

```diff
@@ -19,3 +20,15 @@ The `{{page.slug}}` format represents a date and time as defined by date-time -
 
 * [#{{ page.issue }}](https://github.com/OAI/OpenAPI-Specification/issues/{{ page.issue }})
 {% endif %}
+
```

Comment:
The RFC link on line 16 is broken. The correct link is https://www.rfc-editor.org/rfc/rfc3339#section-5.6

Reply:
fixed